### DISCUSSION TITLE
Adding labels to the manifest

### DISCUSSION THREAD

**Discussion by paresy** [oras-project/oras#316] [Q&A] [answered]
In the Dockerfile i have added this line: 

> LABEL org.opencontainers.image.source="https://github.com/org/repo"

In the manifest it looks the following: 
```
{
  "digest": "...",
  "mediaType": "...",
  "size": 0,
  "config": {
   ...
  },
  "layers": [
    ...
  ],
  "labels": {
    "org.opencontainers.image.source": "https://github.com/org/repo"
  }
}
```

I would like to achieve the same using ORAS, but as far as i have seen i can only add an "annotations" field, but not a "labels" field. Is this correct? Or did i miss something?

Thanks!

**Comment by sajayantony** [comment, marked as answer]
@paresy - I don't believe labels exist in the manifest - https://github.com/opencontainers/image-spec/blob/main/manifest.md
They actually get injected into the config if I am not mistaken. For the manifest we expect the usage of annotations as per the specification.
